Further emendations (#16)
PecanProject/betydb-documentation: about_betydb.md

serves as a front end (also referred to as BETYdb).  The BETYdb database
contains plant trait and yield data that supports research, forecasting, and
decision making associated with agricultural and ecological systems.

> While the original focus of BETYdb was agronomy of cellulosic biofuels, the software has

Author: I think this passage could be written to be a little clearer.

> been adopted for use in ecological forecasting as the core database for the [PEcAn Project](https://pecanproject.org){target="_blank"}

Reviewer: ```suggestion
been adopted for wider use. Specifically, as the core database for the [PEcAn Project](https://pecanproject.org){target="_blank"} crop and ecosystem modeling workflow, it is used for ecological research and forecasting.  
```

> and for high throughput phenotyping (phenotyping = measuring plants) applications in breeding and agronomy trials as the trait database for [TERRA REF](https://terraref.org){target="_blank"} and related projects.

Reviewer: ```suggestion
BETYdb is also the core database for trait and agronomic metadata in high throughput phenotyping (phenotyping = measuring plants) applications, specifically used for breeding and agronomy trials as the trait database for [TERRA REF](https://terraref.org){target="_blank"}, the Drone Processing Pipeline, and related projects. 
There is an [application](https://github.com/terraref/brapi) that exports data from BETYdb in a Breeder's API (BrAPI) compliant interface.
```

If you use or refer to BETYdb data or software in your research, please cite this paper:

> LeBauer, D., Kooper, R., Mulrooney, P., Rohde, S., Wang, D., Long, S. P., & Dietze, M. C. (2018). BETYdb: a yield, trait, and ecosystem service database applied to second‐generation bioenergy feedstock production. _GCB Bioenergy, 10_, 61–71.  [doi:10.1111/gcbb.12420](https://doi.org/10.1111/gcbb.12420){target="_blank"}

In addition, if you are using _data_ from an instance of BETYdb, please

1. Use the citations associated with each record if the publisher will allow

> 2. Refer to the footer of the instance of BETYdb that you are using for citation information.

Reviewer: ```suggestion
2. Archive a copy of the data used in the study. The data in BETYdb is not versioned.  
2. Refer to the footer of the instance of BETYdb that you are using for citation information.
```

   * For example, if you are using data from BETYdb.org, please cite the source of data as:

> > LeBauer, David; Dietze, Michael; Kooper, Rob; Long, Steven; Mulrooney, Patrick; Rohde, Gareth Scott; Wang, Dan; (2010): Biofuel Ecophysiological Traits and Yields Database (BETYdb); Energy Biosciences Institute, University of Illinois at Urbana-Champaign. [http://dx.doi.org/10.13012/J8H41PB9](http://dx.doi.org/10.13012/J8H41PB9){target="_blank"}

Author: The footer of the current version on betydb.org has
> LeBauer, David, Rob Kooper, Patrick Mulrooney, Scott Rohde, Dan Wang, Stephen Long, Michael Dietze, (2017). BETYdb: A yield, trait, and ecosystem service database applied to second‐generation bioenergy feedstock production. GCB Bioenergy. doi:10.1111/gcbb.12420

The citation information given in the example doesn't match this.  But I wasn't quite sure if this information is quite correct either, so I didn't update the example.
